bin/packages/watch.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * External dependencies
+ */
+const fs = require( 'fs' );
+const { execSync } = require( 'child_process' );
+const path = require( 'path' );
+const chalk = require( 'chalk' );
+
+/**
+ * Internal dependencies
+ */
+const getPackages = require( './get-packages' );
+
+const BUILD_CMD = `node ${ path.resolve( __dirname, './build.js' ) }`;
+
+let filesToBuild = new Map();
+
+const exists = ( filename ) => {
+	try {
+		return fs.statSync( filename ).isFile();
+	} catch ( e ) {}
+	return false;
+};
+const rebuild = ( filename ) => filesToBuild.set( filename, true );
+
+getPackages().forEach( ( p ) => {
+	const srcDir = path.resolve( p, 'src' );
+	try {
+		fs.accessSync( srcDir, fs.F_OK );
+		fs.watch( path.resolve( p, 'src' ), { recursive: true }, ( event, filename ) => {
+			const filePath = path.resolve( srcDir, filename );
+
+			if ( ( event === 'change' || event === 'rename' ) && exists( filePath ) ) {
```

Comment:
According to https://nodejs.org/docs/latest/api/fs.html#fs_fs_watch_filename_options_listener `change` and `rename` are the only two possible options for event this can probably be shortened to `exists( filePath )`.